benchmarking/nursery/benchmark_automl/benchmark_definitions.py

```diff
@@ -37,18 +38,34 @@ def fcnet_benchmark(dataset_name):
         mode="min",
         blackbox_name="fcnet",
         dataset_name=dataset_name,
+        max_resource_attr="epochs",
     )
 
 
+NAS201_MAX_WALLCLOCK_TIME = {
+    "cifar10": 5 * 3600,
+    "cifar100": 6 * 3600,
+    "ImageNet16-120": 8 * 3600,
+}
+
+
+NAS201_N_WORKERS = {
+    "cifar10": 4,
+    "cifar100": 4,
+    "ImageNet16-120": 8,
+}
```

Comment:
This changes the original definition, could you let the initial values here and update only those in the dehb benchmark? e.g.:
```suggestion
NAS201_MAX_WALLCLOCK_TIME = {
    "cifar10": 6 * 3600,
    "cifar100": 6 * 3600,
    "ImageNet16-120": 6 * 3600,
}


NAS201_N_WORKERS = {
    "cifar10": 4,
    "cifar100": 4,
    "ImageNet16-120": 4,
}
```